src/main/java/mate/controller/cars/AddDriversToCarController.java

```diff
@@ -0,0 +1,37 @@
+package mate.controller.cars;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.CarServiceImpl;
+import mate.service.DriverService;
+import mate.service.DriverServiceImpl;
+
+public class AddDriversToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarServiceImpl.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverServiceImpl.class);
```

Comment:
```suggestion
    private static final CarService carService =
            (CarService) injector.getInstance(CarService.class);
    private static final DriverService driverService =
            (DriverService) injector.getInstance(DriverService.class);
```
This is actually a hardcore we are trying to avoid with injector)